UnityProject/Assets/Scripts/Furniture/OccupiableDirectionalSprite.cs

```diff
@@ -119,7 +128,7 @@ private void UpdateFrontSprite()
 	{
 		if (spriteRendererFront)
 		{
-			if (occupant > 0)
+			if (occupantNetId > 0)
```

Comment:
Will be preferable to use the NetId.Empty constant instead of 0 (makes it more pretty)